### DISCUSSION TITLE
saf-training/courses/advanced/03

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#297] [Ideas]
# saf-training/courses/advanced/03

Revisiting the NGINX Web Server InSpec Profile In the , we wrote and ran an InSpec profile against a test container. We then generated a report on our results and loaded them in...

https://mitre.github.io/saf-training/courses/advanced/03.html

**Comment by karikarshivani** [comment]
Change `redhat8` running container list output to `redhat9`

**Comment by karikarshivani** [comment]
`inputs-linux.yml` seems to be the name mentioned in the Review section under The Controls, however the exec command uses `inputs.yml` as the name
